tests/test_synop2bufr.py

```diff
@@ -47,31 +47,32 @@ def single_message():
 
 @pytest.fixture
 def metadata_string():
-    md = "station_name,wigos_station_identifier,traditional_station_identifier,facility_type,latitude,longitude,elevation,territory_name,wmo_region\n" + \
-            "OCNA SUGATAG,0-20000-0-15015,15015,Land (fixed),47.77706163,23.94046026,503,Romania,6\n" + \
-            "BOTOSANI,0-20000-0-15020,15020,Land (fixed),47.73565324,26.64555017,161,Romania,6\n" + \
-            "IASI,0-20000-0-15090,15090,Land (fixed),47.16333333,27.62722222,74.29,Romania,6"  # noqa
+    md = "station_name,wigos_station_identifier,traditional_station_identifier,facility_type,latitude,longitude,elevation,barometer_height,territory_name,wmo_region\n" + \
+            "OCNA SUGATAG,0-20000-0-15015,15015,Land (fixed),47.77706163,23.94046026,503,503,Romania,6\n" + \
```

Comment:
504.43 for the barometer height. If we're going to include the information we should make sure it is correct.